packages/admin-panel-server/src/connections/MeditrakConnection.ts

```diff
@@ -2,14 +2,30 @@
  * Tupaia
  * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
  */
-
-import { QueryParameters, ApiConnection } from '@tupaia/server-boilerplate';
+import camelcaseKeys from 'camelcase-keys';
+import { AccessPolicy } from '@tupaia/access-policy';
+import { QueryParameters } from '@tupaia/server-boilerplate';
+import { SessionHandlingApiConnection } from './SessionHandlingApiConnection';
+import { BES_ADMIN_PERMISSION_GROUP } from '../constants';
 
 const { MEDITRAK_API_URL = 'http://localhost:8090/v2' } = process.env;
 
-export class MeditrakConnection extends ApiConnection {
+const isBESAdmin = (policy: Record<string, string[]>) => {
+  return new AccessPolicy(policy).allowsSome(null, BES_ADMIN_PERMISSION_GROUP);
+};
+
+export class MeditrakConnection extends SessionHandlingApiConnection {
   baseUrl = MEDITRAK_API_URL;
 
+  async getUser() {
+    // if user is not logged in, return null rather than fetching the api client user details
+    if (!this.hasSession) {
+      return {};
+    }
```

Comment:
Similar comment in `SessionHandlingApiConnection` class, we probably don't need to do this as users are always required to log in with admin-panel?
```suggestion
```